packages/common/src/components/emoji-picker/emojiPicker.css

```diff
@@ -1,4 +1,4 @@
-@import '../../../../node_modules/emoji-mart/css/emoji-mart.css';
+/* @import '~emoji-mart/css/emoji-mart.css'; */
```

Comment:
We don't use Emoji picker in this PR, I will open a new PR for it and will uncomment this line then.